conceptnet5/vectors/evaluation/analogy.py

```diff
@@ -305,7 +331,7 @@ def eval_google_analogies(vectors, subset='semantic', vocab_size=200000, verbose
         result = best_analogy_3cosmul(
             vectors, tframe, *prompt
         )
-        if result == answer:
+        if result == answer or result in answer:
```

Comment:
This seems extremely forgiving -- isn't this saying that an answer is correct if it's any substring of the correct answer?

Reply:
You're right.